components/system/components/Buttons.js

```diff
@@ -306,3 +306,143 @@ export const ButtonWarning = (props) => {
+  box-shadow: 0 0 0 1px ${Constants.system.bgGray} inset;
+
+  :hover {
+    background-color: #b51111;
+  }
+
+  :focus {
+    outline: 0;
+    border: 0;
+  }
+`;
+
+const STYLES_BUTTON_DELETE_TRANSPARENT = css`
+  ${STYLES_BUTTON}
+  cursor: pointer;
+  background-color: transparent;
+  color: ${Constants.system.darkGray};
+`;
+
+export const ButtonDelete = (props) => {
+  if (props.loading) {
+    return (
+      <button
+        css={props.transparent ? STYLES_BUTTON_DELETE_TRANSPARENT : STYLES_BUTTON_DELETE}
+        style={{ width: props.full ? "100%" : "auto", ...props.style }}
+      >
+        <LoaderSpinner style={{ height: 16, width: 16 }} />
+      </button>
+    );
+  }
+
+  if (props.type === "label") {
+    return (
+      <label
+        css={props.transparent ? STYLES_BUTTON_DELETE_TRANSPARENT : STYLES_BUTTON_DELETE}
+        style={{ width: props.full ? "100%" : "auto", ...props.style }}
+        onClick={props.onClick}
+        children={props.children}
+        type={props.label}
+        htmlFor={props.htmlFor}
+      />
+    );
+  }
+
+  return (
+    <button
+      css={props.transparent ? STYLES_BUTTON_DELETE_TRANSPARENT : STYLES_BUTTON_DELETE}
+      onClick={props.onClick}
+      children={props.children}
+      style={{ width: props.full ? "100%" : "auto", ...props.style }}
+    />
+  );
+};
+
+export const ButtonDeleteFull = (props) => {
+  return <ButtonDelete full {...props} />;
+};
+
+const STYLES_BUTTON_DELETE_DISABLED = css`
+  ${STYLES_BUTTON}
+  cursor: not-allowed;
+  background-color: ${Constants.system.bgRed};
+  color: ${Constants.system.white};
+  box-shadow: 0 0 0 1px ${Constants.system.bgGray} inset;
+
+  :focus {
+    outline: 0;
+    border: 0;
+  }
+`;
+
+const STYLES_BUTTON_DELETE_DISABLED_TRANSPARENT = css`
+  ${STYLES_BUTTON}
+  cursor: not-allowed;
+  background-color: transparent;
+  color: ${Constants.system.gray};
+`;
+
+export const ButtonDeleteDisabled = (props) => {
+  return (
+    <button
+      css={props.transparent ? STYLES_BUTTON_DELETE_DISABLED_TRANSPARENT : STYLES_BUTTON_DELETE_DISABLED}
+      onClick={props.onClick}
+      children={props.children}
+      type={props.label}
+      htmlFor={props.htmlFor}
+      style={{ width: props.full ? "100%" : "auto", ...props.style }}
+    />
+  );
+};
+
+export const ButtonDeleteDisabledFull = (props) => {
+  return <ButtonDeleteDisabled full {...props} />;
+};
+
+const STYLES_BUTTON_CANCEL = css`
+  ${STYLES_BUTTON}
+  cursor: pointer;
+  color: ${Constants.system.black};
+  background-color: ${Constants.system.gray20};
+  box-shadow: 0 0 0 1px ${Constants.system.bgGray} inset;
+
+  :hover {
+    background-color: ${Constants.system.gray30};
+  }
+
+  :focus {
+    outline: 0;
+    border: 0;
+  }
+`;
+
+const STYLES_BUTTON_CANCEL_TRANSPARENT = css`
+  ${STYLES_BUTTON}
+  cursor: pointer;
+  background-color: transparent;
+  color: ${Constants.system.darkGray};
+`;
+
+export const ButtonCancel = (props) => {
```

Comment:
Same goes for here. Let's use ButtonSecondary instead of this for now but leave the code here for future use if we convert over